jwst/regtest/sdp_pools_source.py

```diff
@@ -2,10 +2,12 @@
 # Setup a base class and instantiate it in order to provide the
 # SDP pool lists for test parametrization.
 # #############################################################
-from jwst.tests.base_classes import BaseJWSTTest
+import os
 
+from jwst.regtest.regtestdata import RegtestData
 
-class SDPPoolsSource(BaseJWSTTest):
+
+class SDPPoolsSource(RegtestData):
```

Comment:
Glad to see `BaseJWSTTest` dropped.  Can we now delete that class and maybe that whole module?

Reply:
✅